packages/core/stories/form-field/form-field.stories.tsx

```diff
@@ -37,7 +37,7 @@ export const Skeleton: ComponentStory<typeof FormField> = (props) => {
 
 export const Default: ComponentStory<typeof FormField> = (props) => {
   return (
-    <FlowLayout style={{ width: "366px" }}>
+    <FlowLayout style={{ width: "366px", textAlign: "center" }}>
```

Comment:
I assume these were only for a test?

```suggestion
    <FlowLayout style={{ width: "366px"  }}>
```

Reply:
added just to test its global, but yes can remove